src/index.html

```diff
@@ -0,0 +1,246 @@
+              have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
+            </div>
+          </div>
+        </div>
+        <p class="card_src">lipsum.com</p>
+      </a>
+      <a href="#" class="card">
+        <div class="card__body">
+          <div class="card_image-wrapper">
+            <div class="card_img"></div>
+          </div>
+          <div class="card__icon"></div>
+          <p class="card__warning">Войдите, чтобы сохранять статьи</p>
+          <div class="card_wrapper">
+            <p class="card_date">2 августа, 2019</p>
+            <h4 class="card_title">Why do we use it?</h4>
+            <div class="card_text">It is a long established fact that a reader will be distracted by the readable content
+              of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal
+              distribution of letters, as opposed to using 'Content here, content here', making it look like readable
+              English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model
+              text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions
+              have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
+            </div>
+          </div>
+        </div>
+        <p class="card_src">lipsum.com</p>
+      </a>
+      <a href="#" class="card">
+        <div class="card__body">
+          <div class="card_image-wrapper">
+            <div class="card_img"></div>
+          </div>
+          <div class="card__icon card__icon_marked"></div>
+          <p class="card__warning">Войдите, чтобы сохранять статьи</p>
+          <div class="card_wrapper">
+            <p class="card_date">2 августа, 2019</p>
+            <h4 class="card_title">Why do we use it?</h4>
+            <div class="card_text">It is a long established fact that a reader will be distracted by the readable content
+              of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal
+              distribution of letters, as opposed to using 'Content here, content here', making it look like readable
+              English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model
+              text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions
+              have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
+            </div>
+          </div>
+        </div>
+        <p class="card_src">lipsum.com</p>
+      </a>
+    </div>
+    <button class="button results__button">Показать еще</button>
+  </main>
+  <section class="about results__about">
+    <div>
```

Comment:
No need for that extra div. Use padding instead.
Content in this section is overflowing the page on resolution width less than 1020px.